### PR TITLE
add Julia programming language

### PR DESCRIPTION
**Pull request by msaf9**
### Description

Adding Julia programming language to the docs.

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/julia/julia.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+Title: 'Julia'
+Description: 'Julia stands out as an advanced programming language tailored for technical computing, boasting both high-level functionality and top-notch performance. Its inception aimed to bridge the gap between the user-friendliness of dynamic languages like Python and the swiftness found in statically-typed languages such as C or Fortran.'
```
[the remark this answers is not in the document]
Comment (msaf9) [reply content/julia/julia.md]: I have modified it. Thank you!

**Review by mohitO12-dev — APPROVED** [review]
Approved!

File: content/julia/julia.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+Title: 'Julia'
+Description: 'Julia distinguishes itself as an advanced programming language designed for technical computing, featuring high-level functionality and top-notch performance. Its creation aimed to bridge the gap between the user-friendliness of dynamic languages like Python and the speed found in statically-typed languages such as C or Fortran.'
```
Comment (avdhoottt) [inline content/julia/julia.md:3]: You must follow this [topic-entry-template](https://github.com/Codecademy/docs/blob/main/documentation/topic-entry-template.md) to write the metadata for a new topic. Please include the missing fields.

**Review by avdhoottt — CHANGES_REQUESTED** [review]
Hey @msaf9, the metadata is incomplete, please look into it.

File: content/julia/julia.md (hunk 2 above)
Comment (msaf9) [reply content/julia/julia.md:3]: Thank you for the feedback. I've reviewed the requirements and I understand the need for completeness in the metadata. However, as there is currently no Codecademy Hub Page or Catalog Content available for Julia, I've omitted those sections from the metadata accordingly. Please let me know if there are any further adjustments needed.

**Review by avdhoottt — APPROVED** [review]
LGTM!!

**Pull request merged**